### PR TITLE
Fix/megamenu bg

### PR DESCRIPTION
**Pull request by felix-ico**

### CODE REVIEW

File: packages/components/src/components/telekom/app-header/app-header.tsx [hunk 1]
```diff
@@ -218,11 +218,8 @@ export class Header {
       readData(this.mainNavigation),
       this.activeRouteId
     );
-    const isActive = (item) =>
-      rootNode &&
-      rootNode.id === item.id &&
-      !this.visibleMegaMenu &&
-      this.visibleMegaMenu !== null;
+    const isActive = (item) => rootNode && rootNode.id === item.id;
+    console.log(rootNode, this.visibleMegaMenu);
```
Comment (nowseemee) [inline packages/components/src/components/telekom/app-header/app-header.tsx:222]: 🎯

Comment (maomaoZH) [reply packages/components/src/components/telekom/app-header/app-header.tsx:222]: sry i will remove it

Comment (felix-ico) [reply packages/components/src/components/telekom/app-header/app-header.tsx:222]: this PR includes 2 fixes, one from Maomao and one from me , this code is part of @maomaoZH change :)

Comment (maomaoZH) [reply packages/components/src/components/telekom/app-header/app-header.tsx]: done and pushed

**Comment by nowseemee** [comment]
nice cleanup! unfortunately the rounded separator is removed with these changes. is it intentional? 
image

**Comment by felix-ico** [comment]
> nice cleanup! unfortunately the rounded separator is removed with these changes. is it intentional? image

ah thanks for checking i didn't notice that in light mode, fixing it

**Comment by nowseemee** [comment]
Dark mode was like this: 
image

**Comment by felix-ico** [comment]
ok I added the separator and those "ghost" elements to make the border radius - i had removed too much code previously!

File: packages/components/src/components/telekom/app-header/app-header.tsx [hunk 2]
```diff
@@ -218,11 +218,7 @@ export class Header {
       readData(this.mainNavigation),
       this.activeRouteId
     );
-    const isActive = (item) =>
-      rootNode &&
-      rootNode.id === item.id &&
-      !this.visibleMegaMenu &&
```
Comment (nowseemee) [inline packages/components/src/components/telekom/app-header/app-header.tsx:224]: are you sure this was dead code? you can never know. can we keep it, just to be sure? the header has so many use-cases.

Comment (felix-ico) [reply packages/components/src/components/telekom/app-header/app-header.tsx:224]: @maomaoZH i think this was part of your bugfix, what do you think?

Comment (maomaoZH) [reply packages/components/src/components/telekom/app-header/app-header.tsx:224]: So the designer wants to keep the selected nav status even when user navigates to other tab. If we keep the code, every time when user navigates to other tab which has mega-menu, the selected status will lose.

Comment (maomaoZH) [reply packages/components/src/components/telekom/app-header/app-header.tsx:224]: give me one sec please. dont merge yet

Comment (maomaoZH) [reply packages/components/src/components/telekom/app-header/app-header.tsx:224]: https://user-images.githubusercontent.com/23239723/160800784-c36f829e-5d43-4627-b7e7-901d73e968cc.mov

have you noticed the menu is jumping.

Comment (felix-ico) [reply packages/components/src/components/telekom/app-header/app-header.tsx:224]: I pushed a fix for this, thanks for checking!

**Pull request merged**